templates/compose/tailscale.yaml

```
@@ -0,0 +1,37 @@
+# documentation: https://tailscale.com/kb
+# slogan: Tailscale securely connects your devices over the internet using WireGuard.
+# category: networking
+# tags: vpn, wireguard, remote-access
+# logo: svgs/tailscale.svg
+
+version: '3.7'
+services:
+  tailscale-nginx:
+    image: 'tailscale/tailscale:latest'
+    hostname: '${TS_HOSTNAME:-coolify-ts}'
+    environment:
+      - 'TS_HOSTNAME=${TS_HOSTNAME:-coolify-ts}'
+      - 'TS_AUTHKEY=${TS_AUTHKEY:-your_authkey}'
```

Make the auth key required
```suggestion
      - 'TS_AUTHKEY=${TS_AUTHKEY:?}'
```